src/openzeppelin/account/account.cairo

```diff
@@ -40,17 +40,22 @@ mod Account {
     use array::ArrayTrait;
     use box::BoxTrait;
     use ecdsa::check_ecdsa_signature;
+    use option::OptionTrait;
     use serde::ArraySerde;
```

Comment:
we might want to order these by some criteria, at least i'd group the traits together

Reply:
I've been ordering alphabetically, but as long as we have a criteria I'm ok with refactoring the order.